src/uct/ib/mlx5/dv/ib_mlx5_dv.c

```diff
@@ -157,6 +157,8 @@ ucs_status_t uct_ib_mlx5_devx_create_qp(uct_ib_iface_t *iface,
 
     qp->devx.obj = mlx5dv_devx_obj_create(dev->ibv_context, in, sizeof(in),
                                           out, sizeof(out));
+    ucs_debug("created DevX DCI 0x%x, rdma_wr_disabled=%d", qp->devx.obj,
```

Comment:
it should be after creating the object successfully, and "DCI 0x%x" should print the qp_num
Also sorry but it should be in uct_dc_mlx5_iface_create_dci, not uct_ib_mlx5_devx_create_qp